docs/framework/network-programming/internet-protocol-version-6.md

```diff
@@ -41,7 +41,7 @@ The Internet Protocol version 6 (IPv6) is a new suite of standard protocols for
  For more information, see [IPv6 Addressing](../../../docs/framework/network-programming/ipv6-addressing.md), [IPv6 Routing](../../../docs/framework/network-programming/ipv6-routing.md), [IPv6 Auto-Configuration](../../../docs/framework/network-programming/ipv6-auto-configuration.md), [Enabling and Disabling IPv6](../../../docs/framework/network-programming/enabling-and-disabling-ipv6.md), and [How to: Modify the Computer Configuration File to Enable IPv6 Support](../../../docs/framework/network-programming/how-to-modify-the-computer-configuration-file-to-enable-ipv6-support.md).  
   
 ## References  
- The following are selected RFC documents that you can find at the Internet Engineering Task Force site ([http://www.ietf.org](http://www.ietf.org/)):  
+ The following are selected RFC documents that you can find at the Internet Engineering Task Force site ([https://www.ietf.org](https://www.ietf.org/)):  
```

Comment:
change the last part to say: `[Internet Engineering Task Force (IETF)](https://www.ietf.org/) website.`

The last part is `Internet Engineering Task Force site ([https://www.ietf.org](https://www.ietf.org/))`